IntegrationTests/CombinedConfig_FPGA2/script/makeProject.tcl

```diff
@@ -0,0 +1,170 @@
+create_ip -name MatchProcessor_D2PHIC -module_name MP_D2PHIC -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D2PHID -module_name MP_D2PHID -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D3PHIA -module_name MP_D3PHIA -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D3PHIB -module_name MP_D3PHIB -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D3PHIC -module_name MP_D3PHIC -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D3PHID -module_name MP_D3PHID -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D4PHIA -module_name MP_D4PHIA -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D4PHIB -module_name MP_D4PHIB -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D4PHIC -module_name MP_D4PHIC -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D4PHID -module_name MP_D4PHID -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D5PHIA -module_name MP_D5PHIA -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D5PHIB -module_name MP_D5PHIB -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D5PHIC -module_name MP_D5PHIC -vendor xilinx.com -library hls -version 1.0
+create_ip -name MatchProcessor_D5PHID -module_name MP_D5PHID -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L1L2 -module_name FT_L1L2 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L2L3 -module_name FT_L2L3 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L3L4 -module_name FT_L3L4 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L5L6 -module_name FT_L5L6 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_D1D2 -module_name FT_D1D2 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_D3D4 -module_name FT_D3D4 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L1D1 -module_name FT_L1D1 -vendor xilinx.com -library hls -version 1.0
+create_ip -name TrackBuilder_L2D1 -module_name FT_L2D1 -vendor xilinx.com -library hls -version 1.0
+
+
+# Provide name of top-level HDL (without .vhd extension).
+#set topLevelHDL "SectorProcessor"
+set topLevelHDL "SectorProcessorFull"
+
+# Add HDL for algo
+add_files -fileset sources_1 [glob ../hdl/SectorProcessor.vhd]
+add_files -fileset sources_1 [glob ../hdl/SectorProcessorFull.vhd]
+add_files -fileset sources_1 [glob ../hdl/memUtil_pkg.vhd]
+add_files -fileset sources_1 [glob common/hdl/*.vhd]
+remove_files -fileset sources_1 [glob common/hdl/latency_monitor.vhd]
+remove_files -fileset sources_1 [glob common/hdl/tf_mem_new.vhd]
+
+
+# Add HDL for TB
+add_files -fileset sim_1 [glob ../tb/tb_tf_top.vhd]
+
+# Add utility files
+add_files -fileset utils_1 [glob ./pre.tcl]
```

Comment:
@aryd These pre-synthesis/-simulation scripts shouldn't be needed anymore, since we don't include any external LUT files in the VHDL top module. They were removed for other chains in #329.
```suggestion
```

Reply:
Fixed